pytorch_toolkit/ote/tools/train.py

```diff
@@ -28,6 +32,10 @@ def main():
     arg_converter = build_arg_converter(modules['arg_converter'])
     train_args = arg_converter.convert_train_args(MODEL_TEMPLATE_FILENAME, ote_args)
 
+    # Note that ConfigTransformersEngine is not applied here.
```

Comment:
I guess it is redundant information here.

Reply:
Reword, since now training may makes compression afterwards